### DISCUSSION TITLE
User management [Sep 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6577] [Discord Threads]
> 🙋‍♂️ **nachteulen.coffee**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### I might be failing to understand exact requirement here. New signup's does show up under project collaboration menu, and by default it will be "No Access".  
![image0](https://cdn.discordapp.com/attachments/1145731941174354062/1145737733961162752/Screenshot_2023-08-28_at_8.42.21_PM.png?ex=652d52ea&is=651addea&hm=5be641574c05bedfe9d8803abe9b5b94b8409ed5d650564fbf9a70607e43b418&)  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I don't get to see the new users. the list might be too long  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### there is not paging. I scroll until the end and I don't see the "new user without access"  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I copied the list to a text editor and searched for  "@". It shows me only 100 users. I know I have over 190  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### Copied all that I could reach through scrolling  
![image0](https://cdn.discordapp.com/attachments/1145731941174354062/1145739527927910540/image.png?ex=652d5496&is=651adf96&hm=37bb63967f7574a97c56a5e6784856e2ac8a6816aeb2786456c6242658623395&)  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### also date joining shows "few seconds ago". I imagine you already notice that too. It's not big deal.it's just inacurate.  
#  
> 👨‍🦲 **dstala**  
#### tried adding 200 new users, i could see all of them visible in the collaborator list.    
####    
#### can you share your project information?  
![image0](https://cdn.discordapp.com/attachments/1145731941174354062/1145740780833616032/Screenshot_2023-08-28_at_8.54.28_PM.png?ex=652d55c0&is=651ae0c0&hm=b77731be2bba7a92853b4efe3a9e061926de5e3a2ba5d6e78cd08460960bd315&)  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### Node: **v16.20.1**   
#### Arch: **x64**   
#### Platform: **linux**   
#### Docker: **true**   
#### RootDB: **pg**   
#### PackageVersion: **0.111.0**  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### how did you add them?  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### because for me, new users sign up on their own for the nocodb instance  
#  
> 👨‍🦲 **dstala**  
#### > how did you add them?   
#### script, using API  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### simulating the new users created an account?  
#  
> 👨‍🦲 **dstala**  
#### yes. i manually signed up few users too- worked fine. I was on SQLite. let me check over PG. But shouldn't have any impact IMO  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### Did you also singed more users after having the 200`?  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I created another project and I only have 100 listed without access in the collaboration tab  
#  
> 👨‍🦲 **dstala**  
#### 100 out of 190? ok- thats strange.  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### yes  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### I just created another account and now it's there. Sorry for taking your time. I'll pay close attention in the next days to see if I have something similar.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1145731941174354062)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
